### ISSUE TITLE
Fail to compile openclonk v7.0 in Slackware

### ISSUE THREAD

**Issue by dslackw** [openclonk/openclonk#15]
```
[ 97%] Building CXX object CMakeFiles/openclonk-server.dir/src/platform/C4TimeMilliseconds.cpp.o
[ 97%] Building CXX object CMakeFiles/openclonk-server.dir/src/platform/PlatformAbstraction.cpp.o
[ 97%] Building CXX object CMakeFiles/openclonk-server.dir/src/player/C4Player.cpp.o
[ 97%] Building CXX object CMakeFiles/openclonk-server.dir/src/player/C4PlayerList.cpp.o
[ 98%] Building CXX object CMakeFiles/openclonk-server.dir/src/player/C4RankSystem.cpp.o
[ 98%] Building CXX object CMakeFiles/openclonk-server.dir/src/player/C4ScenarioParameters.cpp.o
[ 98%] Building CXX object CMakeFiles/openclonk-server.dir/src/script/C4AulDebug.cpp.o
[ 98%] Building CXX object CMakeFiles/openclonk-server.dir/src/platform/C4AppT.cpp.o
[ 98%] Building CXX object CMakeFiles/openclonk-server.dir/src/platform/C4StdInProc.cpp.o
[100%] Linking CXX executable openclonk-server
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tgetnum'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tgetent'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tgetstr'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tgoto'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `UP'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `BC'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tputs'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `PC'
/usr/lib/gcc/i586-slackware-linux/5.3.0/../../../libreadline.so: undefined reference to `tgetflag'
collect2: error: ld returned 1 exit status
CMakeFiles/openclonk-server.dir/build.make:4788: recipe for target 'openclonk-server' failed
make[2]: *** [openclonk-server] Error 1
CMakeFiles/Makefile2:359: recipe for target 'CMakeFiles/openclonk-server.dir/all' failed
make[1]: *** [CMakeFiles/openclonk-server.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
```

CONFIGURATION:
http://pastebin.com/cQrw7TvE

**Comment by isilkor** [comment]
As a quick workaround, setting the required libraries manually, like:
```
$ cmake -DREADLINE_LIBRARIES="readline;tinfo"
```
might work. Does Slackware install a pkg-config module for readline? 
Otherwise we'll probably just unconditionally link to tinfo.

**Comment by isilkor** [comment]
Fixed in ec0055f.

**Comment by dslackw** [comment]
Thank you!!!
